devlib/target.py

```diff
@@ -1035,6 +1035,18 @@ def dump_logcat(self, filepath, filter=None, append=False, timeout=30):  # pylin
     def clear_logcat(self):
         adb_command(self.adb_name, 'logcat -c', timeout=30)
 
+    def adb_reboot_bootloader(self):
+        adb_command(self.adb_name, 'reboot-bootloader', timeout=30)
+
+    def adb_root(self):
+        adb_command(self.adb_name, 'root', timeout=30)
```

Comment:
That _resolve_paths only happens for `LinuxTarget`; `AndroidTarget` equivalent is here:
https://github.com/ARM-software/devlib/blob/master/devlib/target.py#L1050

So yes, should be safe (and also necessary -- as the rooted commands would still be passed via 'su' otherwise -- great catch by @bjackman)